CHANGELOG.md

```diff
@@ -1,12 +1,39 @@
-# dbt_quickbooks v0.X.X RELEASE TO BE ANNOUNCED
+# dbt_quickbooks v0.14.0
+New major feature alert! Multicurrency is here!
 
-This pull request includes the following updates:
-## Bug Fix
+## 🚨 Breaking Changes 🚨
```

Comment:
Can we be sure to callout the limitation we encountered with transfer and credit card payment transactions as well? Additionally, we should mention that we are now mapping certain accounts via the currency_id to ensure the proper account is receiving certain transaction lines.

Reply:
The mapping certain accounts via the currency_id should be on line 25 of the PR already, but I updated it to hopefully make it more clear.